axelrod/tests/unit/test_moran.py

```diff
@@ -151,30 +261,15 @@ def test_three_players_with_mutation(self):
     def test_four_players(self):
         players = [axelrod.Cooperator() for _ in range(3)]
         players.append(axelrod.Defector())
-        random.seed(10)
+        axelrod.seed(29)
         mp = MoranProcess(players)
         populations = mp.play()
         self.assertEqual(len(mp), 9)
         self.assertEqual(len(populations), 9)
         self.assertEqual(populations, mp.populations)
         self.assertEqual(mp.winning_strategy_name, str(axelrod.Defector()))
 
-    def test_standard_fixation(self):
```

Comment:
Most of the tests in here were testing multiple things (which I've tried to address). This test also tested that you could pass a different match class and has been removed because I've removed the `match_class` attribute (as you suggested earlier).